modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1198,13 +1198,19 @@ Maximum number of bytes returned in a fetch request.
 
 The strategy used to fulfill fetch requests.
 
+* `polling`: Repeatedly polls every partition in the request for new data. The polling interval is set by <<fetch_reads_debounce_timeout,`fetch_reads_debounce_timeout`>> (deprecated).
+
+* `non_polling`: The backend is signaled when a partition has new data, so it doesn’t need to repeatedly read from every partition in the fetch. Redpanda recommends setting this value for most workloads, as it can improve fetch latency and CPU utilization.
+
+* `non_polling_with_debounce`: This option behaves like non_polling but includes a debounce mechanism with a fixed delay specified by <<fetch_reads_debounce_timeout,fetch_reads_debounce_timeout>> at the start of each fetch. By introducing this delay, Redpanda can accumulate more data before processing, leading to fewer fetch operations and returning larger amounts of data. Enabling this option reduces reactor utilization but may also increase end-to-end latency.
```

Comment:
```suggestion
* `non_polling_with_debounce`: This option behaves like `non_polling`, but it includes a debounce mechanism with a fixed delay specified by <<fetch_reads_debounce_timeout,fetch_reads_debounce_timeout>> at the start of each fetch. By introducing this delay, Redpanda can accumulate more data before processing, leading to fewer fetch operations and returning larger amounts of data. Enabling this option reduces reactor utilization, but it may also increase end-to-end latency.
```